erigon-lib/state/integrity.go

```diff
@@ -146,7 +147,16 @@ func (iit *InvertedIndexRoTx) IntegrityInvertedIndexAllValuesAreInRange(ctx cont
 
 	logEvery := time.NewTicker(30 * time.Second)
 	defer logEvery.Stop()
-	iterStep := func(item visibleFile) error {
+	iterStep := func(item visibleFile) (err error) {
+		defer func() {
+			if r := recover(); r != nil {
+				fileName := "not found"
```

Comment:
`not found` sounds bad - because when I will see it - I will think it's an error from OS. probably you mean `unknown`